kola/tests/misc/selinux.go

```diff
@@ -101,42 +146,61 @@ func SelinuxEnforce(c cluster.TestCluster) {
 
 // SelinuxBoolean checks that you can tweak a boolean in the current session
 func SelinuxBoolean(c cluster.TestCluster) {
-	cmdList := []cmdCheckOutput{
-		{"getsebool virt_use_nfs", true, ".*off"},
-		{"sudo setsebool virt_use_nfs 1", false, ""},
-		{"getsebool virt_use_nfs", true, ".*on"},
-	}
+	seBoolean := "virt_use_nfs"
 
 	m := c.Machines()[0]
 
+	tempBoolState, err := getSelinuxBooleanState(c, m, seBoolean)
+	if err != nil {
+		c.Fatalf(`Failed to gather SELinux boolean state: %v`, err)
+	}
+
+	// construct a regexp that looks like ".*off" or ".*on"
+	tempBoolRegexp := ".*" + tempBoolState.newValue
+	cmdList := []cmdCheckOutput{
+		{"sudo setsebool " + seBoolean + " " + tempBoolState.newValue, false, ""},
+		{"getsebool " + seBoolean, true, tempBoolRegexp},
+	}
+
 	testSelinuxCmds(c, m, cmdList)
 
 	// since we didn't persist the change, should return to default value
-	output := c.MustSSH(m, "getsebool virt_use_nfs")
+	postOut := c.MustSSH(m, "getsebool "+seBoolean)
+	postBool := strings.Split(string(postOut), " ")[2]
 
-	if string(output) != "virt_use_nfs --> off" {
-		c.Fatalf(`The SELinux boolean "virt_use_nfs" is incorrectly configured: want %q, got %q`, "virt_use_nfs --> off", string(output))
+	// newBool[0] contains the original value of the boolean
+	if postBool != tempBoolState.originalValue {
+		c.Fatalf(`The SELinux boolean "%q" is incorrectly configured: wanted %q, got %q`, seBoolean, tempBoolState.originalValue, postBool)
 	}
 }
 
 // SelinuxBooleanPersist checks that you can tweak a boolean and have it
 // persist across reboots
 func SelinuxBooleanPersist(c cluster.TestCluster) {
-	cmdList := []cmdCheckOutput{
-		{"getsebool virt_use_nfs", true, ".*off"},
-		{"sudo setsebool -P virt_use_nfs 1", false, ""},
-		{"getsebool virt_use_nfs", true, ".*on"},
-	}
+	seBoolean := "virt_use_nfs"
 
 	m := c.Machines()[0]
 
+	persistBoolState, err := getSelinuxBooleanState(c, m, seBoolean)
+	if err != nil {
+		c.Fatalf(`Failed to gather SELinux boolean state: %v`, err)
+	}
+
+	// construct a regexp that looks like ".*off" or ".*on"
+	persistBoolRegexp := ".*" + persistBoolState.newValue
+	cmdList := []cmdCheckOutput{
+		{"sudo setsebool -P " + seBoolean + " " + persistBoolState.newValue, false, ""},
```

Comment:
Minor: I generally prefer doing `fmt.Sprintf` for cases where there are multiple variables/strings being shoved together.

Reply:
Easy enough...will address this